test/main.js

```diff
@@ -251,7 +251,7 @@ describe('gulp-csslint', function() {
       stream.end();
     });
 
-    it('should support built-in CSSLint formatters', sinon.test(function(done) {
+    it('should support built-in CSSLint formatters', function(done) {
```

Comment:
This is from me trying to mock out `console`... Does nothing that I can tell, as we manually restore the stub